openshift/Jenkinsfile

```diff
@@ -95,29 +101,32 @@ pipeline {
 				not {
 					buildingTag()
 				}
+			  not {
+					branch 'develop'
+				}
 			}
 			steps {
 				container('oc') {
 					script {
 						sh '''#!/bin/bash
-                        set -e
-                        oc project $OC_PROJECT
-                        oc import-image $BASE_IMG --confirm
-                        oc new-build -l ref=$BRANCH_NAME --strategy=docker --name $APP_NAME --to $APP_NAME:$GIT_SHORT_COMMIT --binary --context-dir="" || echo "Probably already exists, start new build"
-                        sleep 3
-                        oc annotate --overwrite buildconfig/$APP_NAME ref=$BRANCH_NAME shortcommit=$GIT_SHORT_COMMIT
-                        patch-bc-current-ref.sh
-                        oc start-build $APP_NAME --from-dir=. --follow=true --wait=true
-                        '''
+								set -e
+								oc project $OC_PROJECT
+								oc import-image $BASE_IMG --confirm
+								oc new-build -l ref=$BRANCH_NAME --strategy=docker --name $APP_NAME --to $APP_NAME:$GIT_SHORT_COMMIT --binary --context-dir="" || echo "Probably already exists, start new build"
+								sleep 3
+								oc annotate --overwrite buildconfig/$APP_NAME ref=$BRANCH_NAME shortcommit=$GIT_SHORT_COMMIT
+								patch-bc-current-ref.sh
+								oc start-build $APP_NAME --from-dir=. --follow=true --wait=true
+								'''
 					}
 				}
 			}
 		}
 		stage('Deploy INT') {
 			when {
 				anyOf {
-					changeRequest target: 'master'
-					changeRequest target: 'main'
+					changeRequest target: 'develop'//'master'
+					changeRequest target: 'development' // do not build on master prs
```

Comment:
we were going to only deploy to INT and TST after a merge into develop
according to this task: https://meemoo.atlassian.net/browse/AVO-2960